components/scream/scripts/gather_all_data.py

```diff
@@ -50,20 +51,19 @@ def formulate_command(self, machine):
 
         repo = scream_docs_repo if self._scream_docs else scream_repo
 
-        # Compute kokkos location
-        if self._kokkos:
-            kokkos_loc = pathlib.Path(self._kokkos.replace("$machine", machine).replace("$compiler", compiler))
-        else:
-            kokkos_loc = pathlib.Path(scream_repo.parent.parent, "externals", "kokkos")
-
         local_cmd = self._run
         # Do magic replacements here
         local_cmd = local_cmd.\
-            replace("$compiler", compiler).\
-            replace("$kokkos", str(kokkos_loc)).\
+            replace("-cxx $cxx_compiler", cxx_compiler).\
```

Comment:
These magic string replacements don't look right. I think you want:
```
replace("$cxx_compiler", cxx_compiler).\
replace("$f90_compiler", f90_compiler).\
```

For some context, in the help examples above, you have:
```
gather-all-data './scripts/test-all-scream --cxx-compiler $cxx_compiler --f90-compiler $f90_compiler -m $machine'
```
Which utilizes two magic strings `$cxx_compiler` and `$f90_compiler`.

I would confirm that the above use case still works with this PR.

Reply:
Ah, good catch.